SDWebImage/SDWebImagePrefetcher.m

```diff
@@ -15,7 +15,9 @@ @interface SDWebImagePrefetchToken () {
     // These value are just used as incrementing counter, keep thread-safe using memory_order_relaxed for performance.
     atomic_ulong _skippedCount;
     atomic_ulong _finishedCount;
-    atomic_ulong _totalCount;
+    atomic_flag  _isAllFinished;
+    
+    unsigned long _totalCount;
```

Comment:
`_totalCount` only be set when initialize. After that, it only has read operation. So no thread related issue.